rag/rag-base/src/commonMain/kotlin/ai/koog/rag/base/files/DocumentUtils.kt

```diff
@@ -212,3 +212,26 @@ internal suspend fun <Path, Document> DocumentProvider<Path, Document>.charsByPa
 internal suspend fun <Path, Document> DocumentProvider<Path, Document>.textByPath(path: Path): String? {
     return charsByPath(path)?.toString()
 }
+
+/**
+ * Expands a DocumentRange by a number of full lines before and after.
+ * Returns a new range clamped to the content bounds.
+ */
+public fun extendRangeByLines(
+    content: CharSequence,
+    range: DocumentProvider.DocumentRange,
+    linesBefore: Int = 2,
```

Comment:
This looks a bit like a magical number to me. Can you please elaborate in the KDoc, why did you choose `2`?